plugins/auth-node/src/IdentityClient.ts

```diff
@@ -32,27 +35,37 @@ const CLOCK_MARGIN_S = 10;
 export class IdentityClient {
   private readonly discovery: PluginEndpointDiscovery;
   private readonly issuer: string;
-  private keyStore: JWKS.KeyStore;
-  private keyStoreUpdated: number;
+  private keyStore?: GetKeyFunction<JWSHeaderParameters, FlattenedJWSInput>;
+  private endpoint?: URL;
 
   /**
    * Create a new {@link IdentityClient} instance.
    */
-  static create(options: {
-    discovery: PluginEndpointDiscovery;
-    issuer: string;
-  }): IdentityClient {
-    return new IdentityClient(options);
+  static create(
+    options: {
+      discovery: PluginEndpointDiscovery;
+      issuer: string;
+    },
+    cooldownMs = 30000,
+  ): IdentityClient {
+    return new IdentityClient(options, cooldownMs);
   }
 
-  private constructor(options: {
-    discovery: PluginEndpointDiscovery;
-    issuer: string;
-  }) {
+  private constructor(
+    options: {
+      discovery: PluginEndpointDiscovery;
+      issuer: string;
+    },
+    cooldownMs: number,
```

Comment:
is this something we need to expose with this upgrade? The default in jose seems to be 30000. In case there's a good reason to expose it I'd say it should be optional in the options object with some documentation to what it does for users.

Reply:
The exposed cooldown parameter is only needed for testing token expiry. I updated the test so that it works with the default cooldown from the library.

Reply:
Good stuff! 👍